datastacks/tests/unit/pyspark/data_quality/test_utils.py

```diff
@@ -77,6 +77,7 @@ def test_execute_validations(spark, dq_config, datasource_context, data, expecte
 
 @pytest.fixture(scope="session")
 def expectation_results():
```

Comment:
Could we get a negative path unit test, covering an exception being thrown by a great-ex validator? Ensuring that we handle it properly?